reducers/plugins/index.js

```diff
@@ -93,6 +104,22 @@ function postTypes(state = {}, action) {
     }
 }
 
+function mainMenuActions(state = [], action) {
+    switch (action.type) {
+    case ActionTypes.RECEIVED_PLUGIN_MENU_ACTIONS: {
+        if (action.data) {
+            return [...action.data, ...state];
```

Comment:
Do we need anything here to avoid duplicates? Is it possible to receive the actions for a single plugin multiple times?

Reply:
In theory it should only be able to receive it once, but I can add checks to make it more robust